.github/settings.yml

```diff
@@ -18,6 +18,115 @@ repository:
   allow_rebase_merge: true
 
 labels:
+  # sane labeling scheme
+  - name: 'Priority: Critical'
+    color: "#e11d21"
+    board: false
+    priority: 10
+  - name: 'Priority: High'
+    color: "#eb6420"
+    board: false
+    priority: 9
+  - name: 'Priority: Medium'
+    color: "#fbca04"
+    board: false
+    priority: 8
+  - name: 'Priority: Low'
+    color: "#009800"
+    board: false
+    priority: 7
+  - name: 'Status: Abandoned'
+    color: "#000000"
+    board: false
+    priority: 1
+  - name: 'Status: Accepted'
+    color: "#009800"
+    board: true
+    priority: 1
+  - name: 'Status: Proposal'
+    color: "#e11d21"
+    board: true
+    priority: 1
+  - name: 'Status: In Progress'
+    color: "#cccccc"
+    board: true
+    priority: 1
+  - name: 'Status: TBD'
+    color: "#e11d21"
+    board: false
+    priority: 1
+  - name: 'Status: Review Needed'
+    color: "#fbca04"
+    board: true
+    priority: 1
+    description: Needs review from a maintainer
+  - name: 'Status: Revision Needed'
+    color: "#e11d21"
+    board: false
+    priority: 1
+    description: Needs code input from a maintainer
+  - name: 'Status: Doc Needed'
```

Comment:
This is a status that requires that one issue can have more status flags set. That I find confusing.

I think having a flag that indicates that Doc, Test, UI Test or Changlog is still needed is great, but that should not be a status.